.github/workflows/multi-arch-build.yaml

```diff
@@ -157,6 +138,19 @@ jobs:
           DELIMITER
           EOF
 
+      # Encode the $VERSION value in metadata so it's available on
+      # 'latest' image and/or if end-users re-tag the version-tagged image.
+      - name: Append version to LABELS env. var. value for stable images
+        # Only the 'stable' matrix source obtains $VERSION
+        if: matrix.source == 'stable'
+        run: |
+          cat << EOF | tee -a $GITHUB_ENV
+          LABELS<<DELIMITER
+          $LABELS
```

Comment:
This is super ultra freaky.  Is there any way you could combine this with the previous ('Define LABELS') one? 

Reason: (1) it's a little distressing to have two such similar complicated-gobbledygook stanzas; and (2) it isn't entirely clear to me as a reader how/where `$LABELS` will be interpreted. It actually looks to me like it will be interpreted in the shell context of this specific `run`, which makes me wonder more about how this magicky stuff works, because the previous stanza only writes to a file but doesn't actually eval anything, and anyhow I really don't want to learn more about this github workflow stuff. I would really like to just accept the previous stanza, assume it accepts, and then sweep it out of mind. Anyhow, this `$LABELS` here is consuming all of my attention, preventing me from looking at anything else.

Reply:
> This is super ultra freaky.

Welcome to Github Actions!

> it's a little distressing

Yes.  The alternative is to have slightly different looking complicated-gobbledygook stanzas since because the `.version` one is only applicable when `matrix.source == 'stable'.  The only other thing I can think of is to use some dummy/filler value for `$VERSION` when `matrix.source != 'stable'.  But that will also be stress-inducing in different ways :confounded: 

> how this magicky stuff works

STOP! Just STOP! It's Github, we're required to love it!  Please smile, open wide, and graciously accept what is on the spoon, for your "enjoyment" :angry: 

> I really don't want to learn more about this github workflow stuff

Trust me, you don't.  It's a horrible, useless mess, peddled to the masses under a guise of "Innovation" and "Exciting" and "Easy to use!"  I'm really (REALLY) angry with myself for accepting the initial implementation and not insisting we do it in Cirrus with simple bash scripts.

> is consuming all of my attention, preventing me from looking at anything else.

I understand, I do. But I also think it's useful (to users and future re-implementation) to start encoding the stable version into metadata.  Let me stare at it again, with yet-another-cup-o-coffee. Perhaps a less fugly solution will occur to me...

Reply:
...okay, this is the best I can do for now.